doc/sphinx/changes.rst

```diff
@@ -8,6 +8,675 @@ Recent changes
+  - Cumulative record and inductive type declarations can now
+    :ref:`specify<813VarianceDecl>` the  variance of their universes.
+  - Various bugfixes and uniformization of behavior with respect to the use of
+    implicit arguments and the handling of existential variables in
+    declarations, unification and tactics.
+  - New warning for :ref:`unused variables <813UnusedVar>` in catch-all match branches that match
+    multiple distinct patterns.
+  - New warning for `Hint` commands outside sections without a locality
+    attribute, whose goal is to eventually remove the fragile default
+    behavior of importing hints only when using `Require`.
+    The recommended fix is to declare hints as `export`, instead of
+    the current default `global`, meaning that they are imported
+    through `Require Import` only, not `Require`.
+    See the following `rationale and guidelines <https://coq.discourse.group/t/change-of-default-locality-for-hint-commands-in-coq-8-13/1140>`_
+    for details.
+  - General support for :ref:`boolean attributes <813BooleanAttrs>`.
+  - Many improvements to the handling of :ref:`notations <813Notations>`,
+    including number notations, recursive notations and notations with bindings.
+    A new algorithm chooses the most precise notation available to print an expression,
+    which might introduce changes in printing behavior.
+  - Tactic improvements in :tacn:`lia` and its :tacn:`zify` preprocessing step,
+    now supporting reasoning on boolean operators such as `Z.leb` and supporting
+    primitive integers :g:`Int63`.
+  - Typing flags can now be specified :ref:`per-constant / inductive <813TypingFlags>`.
+  - Improvements to the reference manual including updated syntax descriptions that
+    match Coq's grammar in several chapters.
+
+See the `Changes in 8.13`_ section and following sections for the
+detailed list of changes, including potentially breaking changes marked
+with **Changed**.
+
+Coq's documentation is available at https://coq.github.io/doc/v8.13/refman (reference
+manual), and https://coq.github.io/doc/v8.13/stdlib (documentation of
+the standard library). Developer documentation of the ML API is available
+at https://coq.github.io/doc/v8.13/api.
+
+Maxime Dénès, Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+Erik Martin-Dorel has maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+The OPAM repository for Coq packages has been maintained by
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
+contributions from many users. A list of packages is available at
+https://coq.inria.fr/opam/www/.
+
+TODO updated maintainers
+Our current 31 maintainers are Yves Bertot, Frédéric Besson, Tej
```

Comment:
Now 32 maintainers. Our new maintainer is Olivier Laurent (@olaure01). The rest of the list is unchanged.